assets/models/system/Phi-4-cuda-gpu/spec.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Phi-4-cuda-gpu
+version: 1
+path: ./
+tags:
+  foundryLocal: ""
+  license: "MIT"
+  licenseDescription: "This model is provided under the License Terms available at <https://huggingface.co/microsoft/phi-4/blob/main/LICENSE>."
+  author: Microsoft
+  inputModalities: "text"
+  outputModalities: "text"
+  task: chat-completion
+  maxOutputTokens: 2048
+type: custom_model
+variantInfo:
+  parents:
+  - assetId: azureml://registries/azureml/models/Phi-4/versions/7
```

Comment:
Updated to match latest version of pre-existing parent model - Was this the right call?